jgt_tools/check_version.py

```diff
@@ -32,16 +33,24 @@ def find_default_branch():
     return [x for x in remote_info if "HEAD branch" in x][0].split(":")[1].strip()
 
 
-def check_version():
-    """Verify the version is changed if any code files are changed."""
-    default_branch = find_default_branch()
+def check_default_branch(default_branch):
+    """Get changed files and pyproject.toml diff for a branch."""
     changed_files = subprocess.check_output(
         ["git", "diff", default_branch, "--name-only"], universal_newlines=True
     ).splitlines()
-
     pyproject_diff = subprocess.check_output(
         ["git", "diff", default_branch, "pyproject.toml"], universal_newlines=True
     )
+    return changed_files, pyproject_diff
+
+
+def check_version():
+    """Verify the version is changed if any code files are changed."""
+    default_branch = find_default_branch()
+    if os.getenv("GITHUB_ACTIONS") == "true":
+        changed_files, pyproject_diff = check_default_branch(f"origin/{default_branch}")
+    else:
+        changed_files, pyproject_diff = check_default_branch(default_branch)
```

Comment:
I stared at this for a while...
I think it might be simpler to say?
```
branch_to_check = f"origin/{default_branch}" if os.getenv("GITHUB_ACTIONS") == "true" else default_branch
changed_files, pyproject_diff = check_default_branch(branch_to_check)
```
?